### PR TITLE
Naive mentor filtering and setup mentee for algorithm

### PR DESCRIPTION
**Pull request by SylviaZiyuZhang**
Not ready for merging

### CODE REVIEW

File: src/main/java/com/google/sps/data/DatastoreAccess.java
Comment (sagunb) [inline src/main/java/com/google/sps/data/DatastoreAccess.java]: where are the bounds (we wanted to say return no more than 100 results but no less than ... results)

**Review by sagunb — CHANGES_REQUESTED** [review]

File: src/main/java/com/google/sps/data/DatastoreAccess.java
Comment (muditg317) [reply src/main/java/com/google/sps/data/DatastoreAccess.java]: the bound is done a few lines below

**Review by sagunb — DISMISSED** [review]
we can merge this after resolving merge conflicts.

**Review by sagunb — APPROVED** [review]

**Pull request merged**
